net/uacme/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2019 Lucian Cristian <lucian.critian@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uacme
+PKG_VERSION:=1.0.13
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ndilieto/uacme/tar.gz/upstream/$(PKG_VERSION)?
+PKG_HASH:=4b57fbbd3d1783dbdca7f5c999dd35f405fe8ccaf8770a5b57c2eed330bdded6
+
+PKG_MAINTAINER:=Lucian Cristian <lucian.cristian@gmail.com>
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-upstream-$(PKG_VERSION)
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+PKG_CONFIG_DEPENDS:= \
+  CONFIG_LIBCURL_GNUTLS \
+  CONFIG_LIBCURL_OPENSSL \
+  CONFIG_LIBCURL_MBEDTLS
+
+include $(INCLUDE_DIR)/package.mk
+
+
+define Package/uacme
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+libcurl +LIBCURL_OPENSSL:libopenssl +LIBCURL_GNUTLS:libgnutls +LIBCURL_GNUTLS:libtasn1 +LIBCURL_MBEDTLS:libmbedtls
```

Comment:
Except for +LIBCURL_GNUTLS:libtasn , all those LIBCURL_ dependencies are useless. libcurl itself handles it.